scintilla/julia_lexilla_v4.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/lexilla/lexers/LexJulia.cxx b/geany/scintilla/lexers/LexJulia.cxx
+index 98ed670..2425aae 100644
+--- a/lexilla/lexers/LexJulia.cxx
++++ b/geany/scintilla/lexers/LexJulia.cxx
+@@ -40,7 +40,8 @@
+ #include "DefaultLexer.h"
+ 
+ using namespace Scintilla;
+-using namespace Lexilla;
++// Geany still uses Lexilla v4
++//using namespace Lexilla;
+ 
+ static const int MAX_JULIA_IDENT_CHARS = 1023;
+ 
+@@ -148,7 +149,9 @@ public:
+ 		delete this;
+ 	}
+ 	int SCI_METHOD Version() const override {
+-		return lvRelease5;
++		// Geany still uses Lexilla v4
++		//return lvRelease5;
++		return lvIdentity;
+ 	}
+ 	const char * SCI_METHOD PropertyNames() override {
+ 		return osJulia.PropertyNames();
+@@ -173,7 +176,9 @@ public:
+ 		return 0;
+ 	}
+ 
+-	static ILexer5 *LexerFactoryJulia() {
++	// Geany still uses Lexilla v4
```

Comment:
Technically this is wrong, Geany uses Scintilla 3.5 which is before the separation of lexilla

Reply:
corrected